src/Core/Configurations/RuntimeConfigValidator.cs

```diff
@@ -42,6 +42,12 @@ public class RuntimeConfigValidator : IConfigValidator
     // of the form @claims.*** delimited by space character,end of the line or end of the string.
     private static readonly string _claimChars = @"@claims\.[^\s\)]*";
 
+    private static readonly List<DatabaseType> _databaseTypesSupportingCreatePolicy =
```

Comment:
Instead of a list, can we have this as a Set for easier lookup for the Contains operation below?

Reply:
Updated